sphinx/writers/manpage.py

```diff
@@ -75,7 +75,7 @@ def apply(self, **kwargs):
                     if matcher(subnode):
                         node.parent.insert(pos + 1, subnode)
                     else:
-                        newnode = node.__class__('', subnode, **node.attributes)
+                        newnode = node.__class__('', '', subnode, **node.attributes)
```

Comment:
To be exact, this is not annotation fix.
In this else-clause, `subnode` is a Text node. That is implicit, and not understandable from code.

This change explicitly give `subnode` as 3rd argument of new node. It is more explicit and type-safe.